code/modules/power/supermatter/supermatter.dm

```diff
@@ -710,6 +710,13 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	else if(istype(AM, /obj/singularity))
 		return
 	else if(isobj(AM))
+		var/obj/O = AM
+		if(O.resistance_flags & INDESTRUCTIBLE)
+			var/image/causality_field = image(icon, null, "causality_field")
+			add_overlay(causality_field, TRUE)
+			radio.talk_into(src, "Anomalous object has breached containment, emergency causality field enganged to prevent reality destabilization.", engineering_channel)
```

Comment:
Currently, flavor texts regarding objects turning into ash generally happens outside of the `Consume` proc. You can have it handled via returning TRUE or FALSE (depends on whether SM actually consumed the object) from `Consume`, or the `Consume` proc itself can handle the flavor texts.

Reply:
Consume is basically everywhere though and would need the radio code copy and pasted a load

Reply:
the flavour text is a bit confusing. What does this have to do with an "anomalous object breaching containment"